industry_restaurant/data/res_config_settings.xml

```diff
@@ -3,6 +3,11 @@
     <record id="res_config_settings_enable_restaurant" model="res.config.settings">
         <field name="qr_code" eval="1"/>
         <field name="group_product_variant" eval="1"/>
+        <field name="group_project_recurring_tasks" eval="1"/>
+        <field name="pos_use_pricelist" eval="1"/>
+        <field name="group_stock_packaging" eval="1"/>
+        <field name="pos_self_ordering_mode" eval="'mobile'"/>
+        <field name="group_uom" eval="1"/>
```

Comment:
You don't define new one, why do you need it?

Reply:
What are you referring to ?